R/build-articles.R

```diff
@@ -255,11 +255,10 @@ data_articles_index <- function(pkg = ".", call = caller_env()) {
   ))
 
   # Check for unlisted vignettes
-  listed <- sections %>%
-    purrr::map("contents") %>%
-    purrr::map(. %>% purrr::map_chr("name")) %>%
-    purrr::flatten_chr() %>%
-    unique()
+  all_names <- purrr::map(sections, function(section) {
+    purrr::map_chr(section$contents, "name")
```

Comment:
a silly question! could `pluck()` be used here?

Reply:
I don't think so because it's also flattening